final_task/rss_reader/arg.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+def parsargs():
+    parser = argparse.ArgumentParser(description='Getting info from sites')
+
+    parser.add_argument('source', type=str, help='RSS URL')
+
+    parser.add_argument(
+        '--limit',
+        type=int, 
+        help='Limit news topics if this parameter provided'
+    )
```

Comment:
Here and below: it is ok to make that into one line. It will be readable anyway.